docs/langs/python.rst

```diff
@@ -46,7 +46,7 @@ Creating a Module
 =================
 
 Python interacts with the motor controller via Vertiq Module Objects. Currently Vertiq Python API 
-supports 3 types of motors: Vertiq 2306, Vertiq 8108, and Fortiq M42BLS
+supports 3 types of motors: Vertiq 23-06, Vertiq 81-08, and Fortiq M42BLS
```

Comment:
Does John have any thoughts on how we wants to hyphenate Fortiq stuff? Or does he not care about the Fortiq

Reply:
He is "indifferent" to how we hyphenate fortiq, and is ok with how this is now